platform/wayland/cog-utils-wl.h

```diff
@@ -101,6 +104,25 @@ struct _CogWlXkb {
     uint8_t modifiers;
 };
 
+struct _CogWlSeat {
+    struct wl_seat *seat;
+    uint32_t        seat_name;
+    uint32_t        seat_version;
```

Comment:
Funny thing: I just realized that `seat_version` is not used anywhere. Let's leave it be for now, but we could make a follow-up patch later on to remove it :wink:

Reply:
I am preparing another change for supporting multiple seats. I will keep this in mind.